pkgs/tools/admin/jetporch/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, rustPlatform,
+  fetchFromSourcehut,
+  fetchurl, openssl,
+  pkg-config,
+  writeText,
+  ... }:
```

```suggestion
{ lib,
  rustPlatform,
  fetchFromSourcehut,
  openssl,
  pkg-config,
}:
```
`...` is generally not used for package definitions in nixpkgs.